code/modules/clothing/spacesuits/flightsuit.dm

```diff
@@ -155,14 +160,24 @@
 	qdel(part_cap)
 	qdel(part_laser)
 	qdel(part_bin)
-	STOP_PROCESSING(SSfastprocess, src)
+	halt = TRUE
+	sleep(2)
+	//STOP_PROCESSING(SSfastprocess, src)
 	part_manip = null
 	part_scan = null
 	part_cap = null
 	part_laser = null
 	part_bin = null
 	..()
 
+/obj/item/device/flightpack/proc/bullshit_processing_proc()
```

Comment:
Not happening in 3 billion years.
Your own nonsense processing is faster than the crazily efficient load-balancing Master Controller? I think not.